package.json

```diff
@@ -5,11 +5,12 @@
   "author": "JetBrains",
   "license": "SEE LICENSE IN LICENSE.txt",
   "main": "index.js",
+  "module": "index.esm.js",
   "files": [
-    "*/*.svg",
-    "*/*.ico",
+    "*/*.{svg,ico,js}",
```

Comment:
adding `js` wildcard here will publish all our dev scripts here, right? Maybe we don't need this

Reply:
Nope, because of publishing occurs in separate directory where [prepare script](https://github.com/JetBrains/logos/blob/0e68dd09593e3c38f8f2fca4aede4d932372cca5/scripts/prepare.sh#L7) copies only some of js files

Reply:
Okay, cool!